### PR TITLE
ADCM-1844_7 add tests for cluster host page

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/ui_tests/app/page/cluster/page.py [hunk 1]
```diff
@@ -190,3 +201,97 @@ class ClusterConfigPage(ClusterPageMixin):
...
+        CommonTable.Pagination.next_page,
+        CommonTable.Pagination.previous_page,
+    ]
+
+    @allure.step("Click on add host button")
+    def click_add_host_btn(self, is_not_first_host: bool = True):
+        self.find_and_click(ClusterHostLocators.add_host_btn)
+        self.wait_element_visible(HostCreationLocators.block)
+        if is_not_first_host:
+            self.wait_element_visible(HostAddPopupLocators.add_new_host_btn).click()
```
[the remark this answers is not in the document]
Comment (annademina) [reply tests/ui_tests/app/page/cluster/page.py]: thnx! added

File: tests/ui_tests/app/page/cluster/page.py [hunk 2]
```diff
@@ -190,3 +201,97 @@ class ClusterConfigPage(ClusterPageMixin):
...
+
+    @allure.step("Get info about host row")
+    def get_host_info_from_row(
+        self, is_cluster_value: bool = True, row_num: int = 0
```
[the remark this answers is not in the document]
Comment (annademina) [reply tests/ui_tests/app/page/cluster/page.py:238]: added docstring

File: tests/ui_tests/app/page/cluster/page.py [hunk 3]
```diff
@@ -190,3 +201,97 @@ class ClusterConfigPage(ClusterPageMixin):
...
+            state=self.find_child(row, row_elements.state).text,
+        )
+
+    @allure.step("Click on host name in row")
+    def click_on_host_name_in_host_row(self, row):
+        self.find_child(row, ClusterHostLocators.HostTable.HostRow.fqdn).click()
+
+    @allure.step("Click on action in host row")
+    def click_on_action_btn_in_host_row(self, row):
+        self.find_child(row, ClusterHostLocators.HostTable.HostRow.actions).click()
```
[the remark this answers is not in the document]
Comment (annademina) [reply tests/ui_tests/app/page/cluster/page.py]: thnx! added

File: tests/ui_tests/app/page/cluster/page.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Sealwing) [reply tests/ui_tests/app/page/cluster/page.py:238]: from the description, I assume that it is a flag if there is a column on page or not
so maybe call it `table_has_cluster_column`?

[the remark this answers is not in the document]
Comment (annademina) [reply tests/ui_tests/app/page/cluster/page.py]: changed

**Review by Sealwing — APPROVED** [review]

File: pylintrc [hunk 4]
```diff
@@ -51,7 +51,7 @@ confidence=
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 
-disable=missing-docstring,invalid-name,no-self-use,abstract-method,unused-argument,no-else-return,duplicate-code,fixme,no-member,too-few-public-methods,wrong-import-order,useless-import-alias,consider-using-with
+disable=missing-docstring,invalid-name,no-self-use,abstract-method,unused-argument,no-else-return,duplicate-code,fixme,no-member,too-few-public-methods,wrong-import-order,useless-import-alias,consider-using-with,redefined-outer-name
```
Comment (dgusakov) [inline pylintrc:54]: Move added mute to the files. This is a common setup and it is relates to the core code too

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: pylintrc (hunk 4 above)
Comment (annademina) [reply pylintrc]: removed

**Review by dgusakov — APPROVED** [review]

**Review by Sealwing — APPROVED** [review]

**Pull request merged**